docs/python_api/method/clumping.md

```diff
@@ -1,8 +1,11 @@
+---
+title: Clumping
```

Comment:
I know this is just a reflection of the modules, but I'd like to see the API for the LD and distnace based clumping contextualised together. What do you think?